src/parse.rs

```diff
@@ -1520,41 +1521,89 @@ struct InlineEl {
 #[derive(Debug)]
 struct InlineStack {
     stack: Vec<InlineEl>,
+    // lower bounds for
+    // _, non_both, * (mod 3), ** (mod 3), *** (mod 3)
+    // for example an underscore empasis will never match
+    // with any element in the stack with index lowerbounds[0] or below
```

Comment:
I *think* it can match at the lowerbound (the code seems do to so), so this should probably read "will never match with any element in the stack below index lowerbounds[0]".

Reply:
Yup, I believe you are right here.